src/ajax/check_hgvs_dialogue.php

```diff
@@ -0,0 +1,610 @@
+<?php
+/*******************************************************************************
+ *
+ * LEIDEN OPEN VARIATION DATABASE (LOVD)
+ *
+ * Created     : 2022-02-26
+ * Modified    : 2022-04-13
+ * For LOVD    : 3.0-27
+ *
+ * Copyright   : 2004-2022 Leiden University Medical Center; http://www.LUMC.nl/
+ * Programmer  : Ivo F.A.C. Fokkema <I.F.A.C.Fokkema@LUMC.nl>
+ *               L. Werkman <L.Werkman@LUMC.nl>
+ *
+ *
+ * This file is part of LOVD.
+ *
+ * LOVD is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LOVD is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LOVD.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *************/
+
+define('ROOT_PATH', '../');
+require ROOT_PATH . 'inc-init.php';
+require ROOT_PATH . 'inc-lib-variants.php';
+header('Content-type: text/javascript; charset=UTF-8');
+
+
+// Retrieving the transcripts to map to.
+// We are using REQUEST and not GET or POST, because the
+//  input of this script can be both GET and POST.
+$aTranscripts = (empty($_REQUEST['transcripts'])? array() : explode('|', urldecode($_REQUEST['transcripts'])));
+
+// Retrieving the name of the input field.
+$sFieldName = urldecode($_REQUEST['fieldName']);
+
+
+
+if (!($_REQUEST['var'])) {
+    // If the variant is empty, we can simply close the script.
+    exit;
+}
+
+
+
+// Retrieve the reference sequence from the info given through the URL.
+if (strpos($_REQUEST['refSeqInfo'], '-') === false) {
+    // The '-' serves as a communication tool; it tells us that
+    //  the given input was a GB + chromosome. When no '-' is
+    //  found, we know that the input was the reference sequence
+    //  of a transcript.
+    $sType = 'VOT';
+    $sReferenceSequence = urldecode($_REQUEST['refSeqInfo']);
+    global $_DB;
+    $bRefSeqIsSupportedByVV = (
+        'hg' == substr($_DB->query('SELECT id FROM ' . TABLE_GENOME_BUILDS . ' LIMIT 1')->fetchColumn(), 0, 2)
+    );
```

Comment:
I only noticed this today;
- Global isn't necessary because you're not in a function,
- If you want to do a `substr()` on a result from MySQL, you'd better just let the database handle it (see MySQL's `LEFT()`).

Reply:
Fixed!